tests/unit_tests/configuration/test_configurator.py

```diff
@@ -211,6 +212,19 @@ def test_fill_from_environmental_variables(configurator):
     if "SIMTOOLS_CONFIG" in os.environ:
         del os.environ["SIMTOOLS_CONFIG"]
 
+    # using .dotenv files with docker: comments are not removed by docker
+    os.environ["SIMTOOLS_DB_API_PORT"] = "27017 #Port on the MongoDB server"
+    os.environ["SIMTOOLS_DB_SERVER"] = "'abc@def.de' # MongoDB server"
+    configurator.config["db_api_port"] = None
+    configurator.config["db_server"] = None
+    configurator._fill_from_environmental_variables()
+    assert configurator.config["db_api_port"] == 27017
+    assert configurator.config["db_server"] == "abc@def.de"
+    if "SIMTOOLS_DB_API_PORT" in os.environ:
```

Comment:
Why do you need to delete these from the env? Wouldn't they be cleared anyway once the test finishes?

Reply:
You are right - I just following the examples before, where also for the last line the env variables are deleted. I would suggest to leave it at it is, as it makes it easier to add additional lines (without having to think about the previous ones).

Reply:
No problem